third_party/istio-head/generate-manifests.sh

```diff
@@ -16,6 +16,4 @@
 
 source "$(dirname $0)/../library.sh"
 
-generate HEAD "$(dirname $0)" \
-      --set values.pilot.env.PILOT_ENABLE_STATUS=true \
```

Comment:
I missed to delete this in https://github.com/knative-sandbox/net-istio/pull/818
If I should send a separated PR, I do.

This script is not used anyway https://github.com/knative-sandbox/net-istio/blob/f92596ff8ae2b2a2b37def30944bb234561c91b4/hack/update-codegen.sh#L64-L66